ops/docker-sequencer-test/run-ha-test.sh

```
@@ -0,0 +1,1693 @@
+
+    # TC-API-08: 乐观锁版本校验 — old version rejected
+    log_info "--- TC-API-08: 乐观锁版本校验 ---"
+    leader_rpc=$(find_leader_rpc)
+    if [ -n "$leader_rpc" ]; then
+        wait_for_ha_leader 15 || true
+        leader_rpc=$(find_leader_rpc)
+    fi
+    if [ -n "$leader_rpc" ]; then
+        local current_version
+        current_version=$(get_membership_version "$leader_rpc")
+        local stale_version=0  # always stale (version 0 is always old after cluster forms)
```

Shellcheck (0.11.0)

[warning] stale_version appears unused. Verify use (or export if used externally).

(SC2034)